tests/processing/reconstruction/test_e2vid.py

```diff
@@ -4,11 +4,10 @@
 import torch
 
 
-def test_run_e2vid():    # type: ignore
+def test_build_e2vid_shape():    # type: ignore
     num_events = 500
     height, width = 20, 40
     events = basic_utils.generate_events(num_events, height, width, 0.1, 0.24)
     reconstructor = E2Vid((height, width), use_gpu=torch.cuda.is_available())
-    # TODO implement me!
-    # image = reconstructor(events)
-    # assert image.shape == (height, width)
+    image = reconstructor(events)
```

Comment:
@shiba24: was there a reason this was commented?

Reply:
I think there was, but I forgot.. Sure let's include it.